pyproject.toml

```diff
@@ -81,6 +81,7 @@ source = [
 ]
 context = '${CONTEXT}'
 dynamic_context = "test_function"
+relative_files = true
```

Comment:
The coverage test kept on giving this error for either MacOS or Windows:
```
No source for code: '/Users/runner/work/sqlmodel/sqlmodel/docs_src/__init__.py'.
```
This is fixed by using the `relative_files` setting 🎉